test/regress/testlist.txt

```diff
@@ -260,7 +260,9 @@ issue111        normal
 issue115        normal
 issue116        normal
 issue112        normal
-cover1          cover,shell
+cover_stmt      cover,shell
+cover_toggle    cover,shell
+cover_branch    cover,shell
```

Comment:
Please follow the existing convention where tests are named `<topic><number>` so these new tests should be named `cover2` and `cover3`. New tests go at the end of the file.

Reply:
Done